public/firebase-messaging-sw.js

```
@@ -0,0 +1,32 @@
+importScripts('https://www.gstatic.com/firebasejs/11.39.0/firebase-app-compat.js');
+importScripts('https://www.gstatic.com/firebasejs/11.39.0/firebase-messaging-compat.js');
+
+const firebaseConfig = firebase.initializeApp(firebaseConfig);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)